SPECS/python-zstd/python-zstd.spec

```
@@ -0,0 +1,109 @@
+%global pypi_name zstd
+%global zstd_version 1.5.5
+
+Name:           python-%{pypi_name}
+Version:        1.5.5.1
+Release:        2%{?dist}
+Summary:        Zstd Bindings for Python
+
+# original zstd bits are GPL-2.0-or-later OR BSD-2-Clause
+License:        BSD-2-Clause AND (GPL-2.0-or-later OR BSD-2-Clause)
+URL:            https://github.com/sergey-dryabzhinsky/python-zstd
+Source:         %{pypi_source}
+
+# Patches to fix test execution
+Patch:          python-zstd-1.5.5.1-test-external.patch
+
+BuildRequires:  gcc
+BuildRequires:  python3-devel
+BuildRequires:  python3dist(setuptools)
+BuildRequires:  pkgconfig(libzstd) >= %{zstd_version}
+
+%description
+Simple Python bindings for the Zstd compression library.
+
+%package -n     python3-%{pypi_name}
+Summary:        %{summary}
+%{?python_provide:%python_provide python3-%{pypi_name}}
+# The library does not do symbol versioning to fully match automatically on
+# Unfortunately our zstd calls this 'zstd-libs' instead of libzstd
+Requires:       zstd-libs%{?_isa} >= %{zstd_version}
+
+%description -n python3-%{pypi_name}
+Simple Python bindings for the Zstd compression library.
+
+
+%prep
+%autosetup -p1 -n %{pypi_name}-%{version}
+# Remove bundled egg-info
+rm -rf src/%{pypi_name}.egg-info
+# Remove precompiled files
+find . -name '*.pyc' -delete
+# Remove bundled zstd library
+rm -rf zstd/
+# do not test the version matching, we don't really need exact version of
+# zstd here
+rm tests/test_version.py
+sed -i -e '/tests\.test_version/d' setup.py
+sed -i -e '/test_version/d' tests/__init__.py
+
+%build
+%py3_build -- --legacy --external
+
+%install
+%py3_install
+
+%check
+%{__python3} setup.py test
+
+%files -n python3-%{pypi_name}
+%license LICENSE
+%doc README.rst
+%{python3_sitearch}/%{pypi_name}-%{version}-py%{python3_version}.egg-info
+%{python3_sitearch}/%{pypi_name}*.so
+
+%changelog
+* Wed Jan 31 13:55:52 EST 2024 Dan Streetman <ddstreet@ieee.org> - 1.5.5.1-2
```

Please let's keep the changelog format consistent with the rest of the spec files.
```suggestion
* Wed Jan 31 2024 Dan Streetman <ddstreet@ieee.org> - 1.5.5.1-2
```